runtime/compiler/aarch64/codegen/J9TreeEvaluator.cpp

```diff
@@ -323,8 +323,172 @@ J9::ARM64::TreeEvaluator::DIVCHKEvaluator(TR::Node *node, TR::CodeGenerator *cg)
       snippet->gcMap().setGCRegisterMask(0xffffffff);
       }
 
-   cg->evaluate(node->getFirstChild());
+TR::Register *
```

Comment:
The closing `}` of `DIVCHKEvaluator()` is accidentally deleted as the CI Build failure shows.